### DISCUSSION TITLE
EditorTextView can be apply it a package？

### DISCUSSION THREAD

**Discussion by Charliego3** [coteditor/CotEditor#1381] [General]
How does EditorTextView apply it as a package in other apps?

**Comment by 1024jp** [comment]
It is not designed to use in such a way.

**Reply by Charliego3** [reply]
Any plans to make it a standalone package? 😁

**Reply by 1024jp** [reply]
There is no motivation to do that for CotEditor.

**Reply by Charliego3** [reply]
Got it, thank you very much for your reply.
